pegasus/sites.v3/code.org/public/teacher-dashboard/section_progress.haml

```diff
@@ -4,90 +4,92 @@ theme: none
 content-type: text/ng-template
 ---
 
-%link(rel='stylesheet' type='text/css' href='/shared/css/teacher-announcement.css')
+#section-progress-react{'ng-init' => '$emit("section-progress-rendered");', 'ng-show' => 'react_progress'}
+#section-progress-angular{'ng-hide' => 'react_progress'}
+  %link{:rel => 'stylesheet', :type => 'text/css', :href => '/shared/css/teacher-announcement.css'}
```

Comment:
are links meant to be nested like this?

Reply:
Good question. It appeared to work for me, but it might be safer to leave the stylesheet outside of the div.